src/compression/gzip/gzip-dowload.md

```diff
@@ -0,0 +1,35 @@
+## Decompress gzip response for a http request
+
+[![flate2-badge]][flate2] [![reqwest-badge]][reqwest] [![cat-decompression-badge]][cat-decompression]
+
+Donwload a content with [`reqwest`]. </br>
+[`GzEncoder`] is responsible for decompress the response.
```

Comment:
- 'decompress' -> 'decompression'.
- Also the description is a little off. Could you look into how the examples from tar section are described?